js/ui/form/ui.form.js

```diff
@@ -456,6 +458,8 @@ const Form = Widget.inherit({
             items: this.option('items'),
             inOneColumn
         });
+
+        triggerResizeEvent(this.$element().find(`.${TOOLBAR_CLASS}`));
```

Comment:
Why dxResize is called for TOOLBAR_CLASS elements only?
Why dxResize is called for all nested TOOLBAR_CLASS elements?

Reply:
Why dxResize is not called for item' DOM element when label size is changed at https://github.com/DevExpress/DevExtreme/blob/21_1/js/ui/form/ui.form.js#L207

> $labelTexts[i].style.width = maxWidth + 'px';

Reply:
> Why dxResize is called for TOOLBAR_CLASS elements only?
> Why dxResize is called for all nested TOOLBAR_CLASS elements?

Initialy, I called for the all dx-widgets - https://github.com/DevExpress/DevExtreme/pull/16990/commits/99493ee654fff7372ae4207742edf6c7fd7070f7

But after that, I reviewed our [documentation](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#editorType) and found, that it is redundant. It seems, that there is only one widget, which has custom resizing logic.

So, I decided to call it for all toolbars, which are placed inside the form widget

Reply:
> Why dxResize is not called for item' DOM element when label size is changed at

Because, calling the dxresize on the DOM node, doesn't refer to the calling _updateDimensions of the toolbar.
Seems, this events is not propagate to the inner elements.  So, calling the dxresize on the parent nodes will not work.

That's why I call it only on the elements with dx-toolbar class

Reply:
ok, we can make a 'spot' change for toolbar only and at that moment only and reinvestigate it after a new ticket will be posted